flang/lib/Optimizer/Dialect/FIROps.cpp

```diff
@@ -889,20 +889,83 @@ mlir::ParseResult fir::GlobalOp::verifyValidLinkage(StringRef linkage) {
   return mlir::success(llvm::is_contained(validNames, linkage));
 }
 
+//===----------------------------------------------------------------------===//
+// ExtractValueOp
+//===----------------------------------------------------------------------===//
+
+template <bool AllowFields>
+static void appendAsAttribute(OperationState &result,
```

Comment:
Is there any reason to pass `result` if it is not used? If so should there be a TODO comment somewhere?

Reply:
This is left over after some refactoring. I'll remove it.